pymoab-py2-18.04

```diff
@@ -0,0 +1,57 @@
+FROM  ubuntu:18.04
+
+# set timezone
+ENV HOME /root
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+ENV HOME /root
+
+RUN apt-get -y --force-yes update
+RUN apt-get install -y --force-yes \
+    software-properties-common \
+    build-essential \
+    git \
+    cmake \
+    gfortran \
+    libblas-dev \
+    python-pip \
+    liblapack-dev \
+    libhdf5-dev \
+    autoconf \
+    libtool
+
+# need to put libhdf5.so on LD_LIBRARY_PATH
+ENV LD_LIBRARY_PATH /usr/lib/x86_64-linux-gnu
+ENV LIBRARY_PATH /usr/lib/x86_64-linux-gnu
+
+# upgrade pip and install python dependencies
+ENV PATH $HOME/.local/bin:$PATH
+RUN python -m pip install --upgrade pip
+RUN pip install numpy scipy cython tables matplotlib setuptools future pytest
+
+# make starting directory
+RUN cd $HOME \
+  && mkdir opt
+
+# build MOAB
+RUN cd $HOME/opt \
+    && mkdir moab \
+    && cd moab \
+    && git clone https://bitbucket.org/fathomteam/moab \
```

Comment:
clone a single branch with `--depth 1` to make this faster (learning from DAGMC review)

Reply:
Doing this doesn't allow you to checkout the correct branch version in the next line. Should we just always use the master branch? I'd rather set it to a specific version.

Reply:
`git clone --depth 1 https://bitbucket.org/fathomteam/moab -b ${branch}`

here is an example to get a specific branch

Reply:
Never mind- combined the clone w/ only cloning a single branch and that seems to work